airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -196,15 +196,15 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals
         reattach_on_restart: bool = True,
         startup_timeout_seconds: int = 120,
         get_logs: bool = True,
-        image_pull_policy: str = 'IfNotPresent',
+        image_pull_policy: Optional[str] = None,
```

Comment:
Can you add this change to https://github.com/apache/airflow/blob/master/airflow/providers/cncf/kubernetes/CHANGELOG.rst please